src/app/Repositories/ArticleRepository.php

```diff
@@ -143,4 +150,19 @@ private function createArticleImages(
         }
         $this->articleImageModel::insert($params);
     }
+
+    private function createArticleTags(
+        int   $article_id,
+        array $tags
+    )
+    {
+        $params = array();
```

Comment:
ここは短縮構文の
```suggestion
        $params = [];
```
でいいと思います。

Reply:
速度が変わるとかじゃないんですけど、PR TIMES内のコードが`array()`を消して生きたいという流れがあるので。